protocols/luci-proto-wireguard/luasrc/model/cbi/admin_network/proto_wireguard.lua

```diff
@@ -81,10 +81,22 @@ preshared_key = section:taboption(
             "cryptography for post-quantum resistance.")
 )
 preshared_key.password = true
-preshared_key.datatype = "and(base64,rangelength(44, 44))"
+preshared_key.datatype = "and(base64,rangelength(44,44))"
 preshared_key.optional = true
 
 
+fwmark = section:taboption(
+  "advanced",
+  Value,
+  "fwmark",
+  translate("Firewall Mark"),
+  translate("Optional. 32-bit hexadecimal mark for outgoing packets. " ..
+            "Enter value without leading <code>0x</code>.")
```

Comment:
Well, I know that. Problem is, it doesn't fit into what we have regarding datatypes in `cbi/datatypes.lua` (source: `modules/luci-base/luasrc/cbi/datatypes.lua`). So the question here is not what input format is the best, but what input format is the best we can achieve the given datatypes. Or, do we want to create a special new datatype for this sort of input, although the majority of users may never need this datatype. Datatypes are global :/